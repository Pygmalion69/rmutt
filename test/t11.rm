// packages
package lesson;

sentence: o " starts with the letter 'O', " greeting.o;
o: "oatmeal", "ogre";

package greeting;

s: "hello there " o;
o: "beautiful", "Mr. Smarty Pants";