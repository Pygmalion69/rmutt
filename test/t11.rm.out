ogre starts with the letter 'O', Mr. Smarty Pants
oatmeal starts with the letter 'O', Mr. Smarty Pants
ogre starts with the letter 'O', beautiful
oatmeal starts with the letter 'O', beautiful
